### DISCUSSION TITLE
Destroying Character Virtual

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#101] [Q&A] [answered]
What would be the correct way to destroy a CharacterVirtual instance? I thought we should do it via body interface, but it seems there is no suitable method for it. Should we just `Jolt.destroy()` it?

**Comment by jrouwe** [comment, marked as answer]
It's a refcounted class so either you do `character.AddRef()` after creation and `character.Release()` to destroy it or if nothing has taken a reference (Jolt currently doesn't take a reference anywhere) then you can also use `Jolt.destroy`.
